src/globals/steam-client/Input.ts

```diff
@@ -560,6 +560,60 @@ export enum EControllerRumbleSetting {
     On,
 }
 
+export enum ControllerInputGamepadButton {
```

Comment:
Initially, the name was `GamepadButton` but it `@decky/ui` already have an enum with the same name.